app/controllers/application_controller.rb

```diff
@@ -120,8 +120,8 @@ def check_for_pdf
   end
 
   def set_host_for_local_storage
-    Rails.application.routes.default_url_options[:host] = request.base_url if Rails.application.config.active_storage.service == :local
+    Rails.application.routes.default_url_options[:host] = request.base_url
```

Comment:
Could this go in `environment.rb` seeing as it no longer depends on the environment? It technically is a config setting.

Reply:
yeah I guess we could. I haven't tested it myself but feel free to give this a go. 
Also probably a matter of making sure the host url to use is pulling the correct one from the env vars